README.md

```diff
@@ -25,7 +25,7 @@ pyPESTO features include:
 * Parameter estimation pipeline for systems biology problems specified in
   [SBML](http://sbml.org/) and [PEtab](https://github.com/PEtab-dev/PEtab)
   ([example](https://github.com/ICB-DCM/pyPESTO/blob/master/doc/example/petab_import.ipynb))
-* Parameter estimation with qualitative data as described in
+* Parameter estimation with ordinal data as described in
   [Schmiester et al. (2020)](https://doi.org/10.1007/s00285-020-01522-w).
   This is currently implemented in the `feature_ordinal` branch.
```

Comment:
```suggestion
```

Branch no longer exists.

Reply:
I should add this too, right? 
([example](https://github.com/ICB-DCM/pyPESTO/blob/master/doc/example/example_ordinal.ipynb))
Possibly commented out until it's in the main (master) branch?

Reply:
Yeah, adding the example is good. Fine with me to have a dead link on `develop` until the next release.

Reply:
Ok, will add it too.